src/style.css

```diff
@@ -1,3 +1,109 @@
 body {
   margin: 0;
+  font-family: "Roboto", "Helvetica", sans-serif;
+}
+
+.products {
```

Comment:
apply padding for all card sides to avoid extra margin for inner elements
Previous mentor asked as well
https://github.com/mate-academy/layout_product-cards/pull/3531#pullrequestreview-1597730871